### PR TITLE
Add missing GD small suite

### PR DESCRIPTION
**Pull request by fenek**
I forgot to add this suite to PR #1763 :(

### CODE REVIEW

File: test/mod_global_distrib_SUITE.erl [hunk 1]
```diff
@@ -0,0 +1,175 @@
+%%==============================================================================
+%% Copyright 2014 Erlang Solutions Ltd.
```
Comment (arcusfelis) [inline test/mod_global_distrib_SUITE.erl]: year :D

Comment (fenek) [reply test/mod_global_distrib_SUITE.erl]: Hahaha :D Copy & paste issues. :D Thanks.

File: test/mod_global_distrib_SUITE.erl [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    fake_stop(),
+    unset_meck(),
+    Config.
+
+%%--------------------------------------------------------------------
+%% Hook handlers tests
+%%--------------------------------------------------------------------
+
+
+%% missing_struct_ tests verify the behaviour of packet_to_component handler,
```
Comment (arcusfelis) [inline test/mod_global_distrib_SUITE.erl]: good :)

File: test/mod_global_distrib_SUITE.erl [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+-author('piotr.nosek@erlang-solutions.com').
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("exml/include/exml.hrl").
+
+%%--------------------------------------------------------------------
+%% Suite configuration
+%%--------------------------------------------------------------------
+
+all() ->
```
Comment (arcusfelis) [inline test/mod_global_distrib_SUITE.erl]: we can list cases in all() directly.
I don't see any point of having groups for small suites.
(the only point is that we use groups everywhere).

File: test/mod_global_distrib_SUITE.erl [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+    % The handler must not crash and return unchanged Acc
+    Acc = mod_global_distrib_mapping:packet_to_component(Acc, From, To).
+        
+%% Update logic has two separate paths: when a packet is sent by a user or by another
+%% component. This test covers the latter.
+missing_struct_in_message_from_component(_Config) ->
+    From = jid:make(<<"">>, <<"from_service.", (global_host())/binary>>, <<"">>),
+    {Acc, To} = fake_acc_to_component(From),
+    % The handler must not crash and return unchanged Acc
+    Acc = mod_global_distrib_mapping:packet_to_component(Acc, From, To).
```
Comment (arcusfelis) [inline test/mod_global_distrib_SUITE.erl]: I am usually using `?assert_equal`.
Recently started to use 
`?assert_equal_extra(A, B, #{comment => "Do you have something to say?", some_var => X, some_other_var => Y})`
which contains enough info to figure out what is wrong, without reading logs or suite code in 50% of cases.

**Pull request merged**